tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-    "include": ["src"],
+    "include": ["src", "test"],
```

Comment:
Unless there's a strong reason I don't think we should include test here.

Reply:
I Included "test" in here to get rid of TS Errors on my test files, but it's not a big deal, I can remove it

Reply:
My bad - Alberto confirmed this is ok [here](https://github.com/balancer/b-sdk/pull/111#discussion_r1370192599)